server/src/main/java/io/crate/auth/AccessControlImpl.java

```diff
@@ -232,6 +233,15 @@ public Void visitQueriedSelectRelation(QueriedSelectRelation relation, RelationC
             for (var source : relation.from()) {
                 source.accept(this, context);
             }
+            for (var symbol : relation.outputs()) {
+                for (var rel : SymbolVisitors.extractAnalyzedRelations(symbol)) {
+                    rel.accept(this, context);
+                }
+            }
+            for (var rel : SymbolVisitors.extractAnalyzedRelations(relation.where())) {
+                rel.accept(this, context);
+            }
```

Comment:
I checked what else could be used in relation and not covered here.

I think `group by/having` could be skipped here as we must(?) have them in `relation.outputs()` which is covered now.

Not sure about` relation.orderBy()` - I think we can do smth like `ORDER BY some_scalar(forbidden_column)`?

Reply:
Yep orderBy and having where also problematic, fixed and added tests.
Group by as you said is in outputs anyway.

Reply:
Actually groupby is also an issue, fixed and added this weird test: https://github.com/crate/crate/pull/14806/files#diff-3d7b12b860814c898cb92651d4a01284f610f277e0503a8e259df2ae773aeffaR399 :)

Reply:
thx again for prompting to check all other possibilities.